VibeSomeMauiBro/Services/CatService.cs

```diff
@@ -0,0 +1,107 @@
+using System.Text.Json;
+using VibeSomeMauiBro.Models;
+
+namespace VibeSomeMauiBro.Services;
+
+public interface ICatService
+{
+    Task<List<Cat>> GetCatsAsync(int count = 10);
+    Task<List<Cat>> GetLikedCatsAsync();
+    Task LikeCatAsync(Cat cat);
+    Task DislikeCatAsync(Cat cat);
+}
+
+public class CatService : ICatService
+{
+    private readonly HttpClient _httpClient;
+    private readonly List<Cat> _likedCats = new();
+    private readonly List<string> _seenCatIds = new();
+
+    public CatService(HttpClient httpClient)
+    {
+        _httpClient = httpClient;
+    }
+
+    public async Task<List<Cat>> GetCatsAsync(int count = 10)
+    {
+        try
+        {
+            // Using The Cat API - free tier allows 10 requests per minute
+            var response = await _httpClient.GetStringAsync($"https://api.thecatapi.com/v1/images/search?limit={count}&has_breeds=1");
```

Comment:
Write a new document in `docs/CatApi.md` that explains this web service.

Following their docs at: https://thecatapi.com/